docs/embedding/sdk/introduction.md

```diff
@@ -30,6 +30,13 @@ Here's the [Shoppy source code](https://github.com/metabase/shoppy).
 - [Quickstart CLI](./quickstart-cli.md) (If you have an app, but no Metabase)
 - [Quickstart with sample React app](./quickstart-with-sample-app.md) (If you don't have either)
 
+## Architecture
```

Comment:
I would put this section below the installation section. This info is more an FYI that not all devs will care about; most devs will just hunt for the command they need to run to get started.

Reply:
Let's move under `installation` because know the architecture before the `development` is better)

Reply:
ok, that's fair